### DISCUSSION TITLE
Should the force option be mergeable?

### DISCUSSION THREAD

**Discussion by fgreinacher** [renovatebot/renovate#25253] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Latest

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We just ran into an interesting issue where we have `force.constraints` in our repository config and additionally tried to pass `force.schedule` via environment variable. As the `force` option is not marked as mergeable the repository-level `force` will completely override the environment variable `force`.

Is this desired behavior? If not I'll be happy to provide a self-contained reproduction.

### What you tried so far.

See above

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
I guess it can be/should be mergeable. When you pass `force.schedule` via environment is your intention to override/disallow custom schedules in every repo?

**Reply by fgreinacher** [reply]
> When you pass force.schedule via environment is your intention to override/disallow custom schedules in every repo?

Yes, we need a way to force-renovate a repository, and it seems configuring `force.schedule` is the best way to do that.

**Reply by fgreinacher** [reply]
I opened https://github.com/renovatebot/renovate/pull/25256, let's continue there :)

**Reply by viceice** [reply]
force isn't allowed on repo config.  do you mean the global bot config file?

**Reply by viceice** [reply]
- https://github.com/renovatebot/renovate/pull/24561

force on repo config will be an error soon

**Reply by fgreinacher** [reply]
We had it in repo config, bot config file and bot env. Good to know repo config won't be possible anymore soon, making it mergeable still makes sense I'd say.

**Reply by rarkins** [reply]
I'm not sure we enforce that today?

**Reply by fgreinacher** [reply]
It is working at the repository level at the moment, it's even documented: https://docs.renovatebot.com/configuration-options/#constraints

**Reply by viceice** [reply]
we have a plan and a open PR to enforce block global only option usage at repo level.

so the question is: allow force on repo level?

**Reply by fgreinacher** [reply, marked as answer]
This has been implemented in https://github.com/renovatebot/renovate/pull/25256
